hadoop-hdds/docs/content/interface/ReconApi.md

```diff
@@ -774,6 +774,114 @@ Returns a summary of the current state of the Ozone cluster.
      }
 ```
 
+## Volumes (admin only)
+
+### GET /api/v1/om/volumes
```

Comment:
I think we can have /api/v1/volumes instead of having om in path to align similar to other resources

Reply:
Thank you for the review. I have removed the /om from the path. PTAL. Thank you.